index.bs

```diff
@@ -5332,9 +5333,9 @@ partial interface MLGraphBuilder {
 
 <details open algorithm>
   <summary>
-    The <dfn method for=MLGraphBuilder id=softmax-noargs>softmax()</dfn> method steps are:
+    The <dfn method for=MLGraphBuilder>softmax(|axis|)</dfn> method steps are:
   </summary>
-    1. Let |op| be the result of [=creating an MLActivation=] given [=this=] and  "softmax".
+    1. Let |op| be the result of [=creating an MLActivation=] given [=this=], "softmax", and «[ "axis" → |axis| ]».
```

Comment:
> «[ "axis" → |axis| ]»

Should we add this [ordered map](https://infra.spec.whatwg.org/#ordered-map) literal into Coding Conventions?